cql3/prepare_context.hh

```diff
@@ -66,14 +67,17 @@ public:
 
     std::vector<lw_shared_ptr<column_specification>> get_variable_specifications() &&;
 
-    std::vector<uint16_t> get_partition_key_bind_indexes(const schema& schema) const;
+    std::vector<uint16_t> get_partition_key_bind_indexes() const;
 
     void add_variable_specification(int32_t bind_index, lw_shared_ptr<column_specification> spec);
 
     void set_bound_variables(const std::vector<shared_ptr<column_identifier>>& bind_variable_names);
 
     void clear_pk_function_calls_cache();
 
+    // Required if set_bound_variables() is used
```

Comment:
Are you sure that's the right reference? It seems get_partition_key_bind_indexes() the function which uses _schema?

Reply:
They're used in combination: set_bound_variables() is used to indicate bind variables, and get_partition_key_bind_indexes() is used to exrtact routing key information from them. I agree the comment needs improvement.